etl/steps/data/garden/wb/2025-05-12/enterprise_surveys.meta.yml

```diff
@@ -0,0 +1,189 @@
+          description_key:
+              - Measures the share of firms that believe companies like theirs are expected to pay informal payments or gifts to secure government contracts
+              - Captures perceived expectations rather than direct reports of bribe payments
+              - Data are collected through structured interviews with private sector firms using stratified random sampling
+              - Bribery in procurement increases business costs reduces fairness and deters competition from firms unwilling or unable to pay
+              - Regional and global averages are computed by taking a simple average of country-level results using only the latest available survey data per country since 2014 following the Enterprise Surveys Global Methodology
+          presentation:
+            title_public: Share of firms expected to bribe secure government contracts
+          display:
+            name: Government contract
+
+      percent_of_firms_expected_to_give_gifts_to_get_an_operating_license:
+          title: Share of firms expected to give bribes to obtain an operating license
+          description_short: Percent of firms expected to give bribes to get an operating license.
+          description_from_producer: |-
+            Percentage of firms reporting that they were expected to provide gifts or informal payments to obtain an operating license. Refusals are treated as affirmative.
+          description_key:
+              - Measures the percentage of firms that reported being expected to pay informal payments to obtain an operating license
+              - Refusal to answer is treated as an affirmative response indicating bribery occurred
+              - Data are collected through structured interviews with private sector firms using stratified random sampling
+              - Corruption in licensing increases delays costs and risks especially for smaller and newer firms
+              - Regional and global averages are computed by taking a simple average of country-level results using only the latest available survey data per country since 2014 following the Enterprise Surveys Global Methodology
+          presentation:
+            title_public: Share of firms expected to give bribes to obtain an operating license
+          display:
+            name: Operating license
+
+      percent_of_firms_expected_to_give_gifts_to_get_an_import_license:
+          title: Share of firms expected tto give bribes to obtain an import license
+          description_short: Percent of firms expected to give bribes to get an import license.
+          description_from_producer: |-
+            Percentage of firms reporting that they were expected to provide gifts or informal payments to obtain an import license.
+          description_key:
+              - Measures the percentage of firms that reported being expected to pay informal payments to obtain an import license
```

Comment:
Maybe it's worth explaining (with a DoD or an additional point) what an import license is.